lib/graphql/input_object_type.rb

```diff
@@ -84,6 +84,7 @@ def coerce_result(value, ctx = nil)
 
     def coerce_non_null_input(value, ctx)
       input_values = {}
+      arg_default_used = {}
```

Comment:
Woops, I'll need to update the naming of things between `used_default` and `default_used` for consistency.